configs/config.pcengines_apu1

```diff
@@ -12,3 +12,4 @@ CONFIG_PAYLOAD_CONFIGFILE="$(top)/src/mainboard/$(MAINBOARDDIR)/seabios_config"
 CONFIG_SEABIOS_REVISION=y
 CONFIG_SEABIOS_REVISION_ID="rel-1.11.0.4"
 CONFIG_SEABIOS_DEBUG_LEVEL=0
+CONFIG_MEMTEST_MASTER=y
```

Comment:
@miczyg1 but apu1 doesn't work with memtest at this point, right?

Reply:
@pietrushnic yes,do You suggest trying older stable commit?